teemaderegister-react/src/components/ViewAllUsers.jsx

```diff
@@ -0,0 +1,37 @@
+import React, {useEffect, useState} from 'react'
+import { Table } from 'antd'
+import API from '../utils/Api'
+import {ADMIN_VIEW_USERS_URL} from '../constants/ApiConstants'
+
+const columns = [
+  {
+    title: 'First Name',
+    dataIndex: 'firstName',
+    key: 'firstName'
+  },
+  {
+    title: 'Last Name',
+    dataIndex: 'lastName',
+    key: 'lastName'
+  }
+]
+
+export default function ViewAllUsers () {
+  const [users, setUsers] = useState([])
+  const [isLoading, setIsLoading] = useState(true)
+
+  useEffect(() => {
+    API('GET', ADMIN_VIEW_USERS_URL)
+      .then(res => {
+        const userInfo = res.map((userData) => ({
```

Comment:
I would pass userdata directly to setUsers

Reply:
Without mapping the response or how?